### PR TITLE
Release: Claim liquid fish after sale

### PR DESCRIPTION
**Pull request by creed-victor**
PR is for FISH token pre-sale, do not merge without mainnet contracts!

### CODE REVIEW

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/**
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx:1]: Delete this comment pls.

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+import { TxType } from 'store/global/transactions-store/types';
+import { useCacheCallWithValue } from 'app/hooks/useCacheCallWithValue';
+import { TxDialog } from 'app/components/Dialogs/TxDialog';
+import { weiToNumberFormat } from '../../../../../utils/display-text/format';
+import { useMaintenance } from 'app/hooks/useMaintenance';
+import { ErrorBadge } from 'app/components/Form/ErrorBadge';
+import { discordInvite } from 'utils/classifiers';
+
+interface Props {
+  className?: object;
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx:25]: Why is this `object`? Usually, it's `string`.

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  address: string;
+}
+export function ClaimForm({ className, address }: Props) {
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx:28]:
```suggestion
export const ClaimForm: React.FC<IClaimFormProps> = ({ className, address }) => {
```

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+import { useSendContractTx } from '../../../../hooks/useSendContractTx';
+import { TxType } from 'store/global/transactions-store/types';
+import { useCacheCallWithValue } from 'app/hooks/useCacheCallWithValue';
+import { TxDialog } from 'app/components/Dialogs/TxDialog';
+import { weiToNumberFormat } from '../../../../../utils/display-text/format';
+import { useMaintenance } from 'app/hooks/useMaintenance';
+import { ErrorBadge } from 'app/components/Form/ErrorBadge';
+import { discordInvite } from 'utils/classifiers';
+
+interface Props {
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx:24]:
```suggestion
interface IClaimFormProps {
```

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  const { value: balance } = useCacheCallWithValue(
+    'lockedFund',
+    'getWaitedUnlockedBalance',
+    '0',
+    address,
+  );
+
+  const unlockTime = useMemo(() => Number(getWaitedTS) * 1000, [getWaitedTS]);
+
+  const handleSubmit = () => {
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx:53]: Use `useCallback`.

File: src/app/pages/OriginsClaimPage/index.tsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/index.tsx:1]: Delete the comment pls.

**Review by tiltom — CHANGES_REQUESTED** [review]

File: src/app/pages/OriginsClaimPage/index.tsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { translations } from 'locales/i18n';
+import { Helmet } from 'react-helmet-async';
+import { Header } from '../../components/Header';
+import { Footer } from '../../components/Footer';
+import { ClaimForm } from './components/ClaimForm';
+import { useAccount } from 'app/hooks/useAccount';
+
+export function OriginsClaimPage() {
```
Comment (tiltom) [inline src/app/pages/OriginsClaimPage/index.tsx:16]:
```suggestion
export const OriginsClaimPage: React.FC = () => {
```

**Review by tiltom — CHANGES_REQUESTED** [review]

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx (hunk 2 above)
Comment (creed-victor) [reply src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx]: Oh, copied component without checking on it :|

**Review by tiltom — DISMISSED** [review]

File: src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx (hunk 2 above)
Comment (tiltom) [reply src/app/pages/OriginsClaimPage/components/ClaimForm/index.tsx]: No problem :)

**Pull request merged**